src/app/view/components/main-legacy/main-legacy.component.html

```diff
@@ -20,3 +20,4 @@
 <dot-large-message-display></dot-large-message-display>
 <dot-alert-confirm></dot-alert-confirm>
 <dot-push-publish-dialog></dot-push-publish-dialog>
+<dot-dot-download-bundle-dialog></dot-dot-download-bundle-dialog>
```

Comment:
We need to find a way to strop adding things here, we need to load this on the fly.